src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -231,44 +211,36 @@ class ItemPage extends React.Component {
               aria-relevant="additions removals"
               aria-describedby="results-description"
             >
-              <div className="nypl-results-item">
+              <div className="nypl-item-details">
                 <h1>{title}</h1>
-
-                <ul className="nypl-item-toc">
-                  <li><a href="#item-holdings">Item holdings</a></li>
-                  <li><a href="#item-details">Item details</a></li>
-                  <li><a href="#item-external-links">External links</a></li>
-                </ul>
-
-                <div id="nypl-results-text">
-                  <ItemOverview
-                    data={itemOverview}
-                  />
-                </div>
-
-                <ItemHoldings
-                  path={this.props.location.search}
-                  holdings={holdings}
-                  title={`${record.numAvailable} item${record.numAvailable === 1 ? '' : 's'} associated with this record:`}
-                />
-
-                <div className="item-details">
-                  <div id="item-details">
-                    <ItemDetails
-                      data={itemDetails}
-                      title="Item details"
-                    />
-                  </div>
-
-                  <div id="item-external-links">
-                    <ItemDetails
-                      data={externalLinks}
-                      title="External links"
-                    />
-                  </div>
+                <div className="nypl-item-info">
+                  <p>
+                    <span className="nypl-item-media">{materialType}</span>
+                    {`${language ? ' in ${language}' : ''}`}
+                  </p>
+                  <p>{record.extent} {record.dimensions}</p>
+                  <p>
+                    {record.placeOfPublication}
+                    {record.publisher} {yearPublished}
+                  </p>
+                  <p>{`${location ? '${location} (how to get there?)' : ''}`}</p>
+                  <p className="nypl-item-use">{usageType}</p>
                 </div>
+              </div>
+            </div>
+            <div className="nypl-column-one-quarter nypl-item-holdings">
+              <ItemHoldings
+                path={this.props.location.search}
+                holdings={holdings}
+                title={`${record.numAvailable} item${record.numAvailable === 1 ? '' : 's'} associated with this record:`}
+              />
+            </div>
 
-              <ItemEditions title={title} item={record} />
+            <div className="nypl-column-three-quarters">
+              <div className="nypl-item-details">
+                  <ItemDetails
```

Comment:
I've been breaking React components into multiple lines if there are more than one attribute. Since there's only one attribute here, it's fine to make it on the same line.